cmd/completion.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+func newCompletionCmd() *cobra.Command {
+	completionCmd := &cobra.Command{
+		Use:   "completion",
+		Short: "Generators for shell completions",
```

Comment:
```suggestion
		Short: "Generators for shell completions",
		Long:`Output shell completion code for the specified shell (bash or zsh). The shell code must be evaluated to provide interactive completion of kubebuilder commands.  This can be done by sourcing it from the .bash_profile.

 Detailed instructions on how to do this are available here:
<add a link for the kubebuilder docs with the further information, note that it should similar to https://kubernetes.io/docs/tasks/tools/install-kubectl/#enabling-shell-autocompletion>`
```